docs/modules/indexes/vectorstores/examples/azuresearch.ipynb

```diff
@@ -0,0 +1,245 @@
+{
+ "cells": [
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Azure Cognitive Search"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Install Azure Cognitive Search SDK"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!pip install --index-url=https://pkgs.dev.azure.com/azure-sdk/public/_packaging/azure-sdk-for-python/pypi/simple/ azure-search-documents==11.4.0a20230509004\n",
+    "!pip install azure-identity"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Import required libraries"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import os, json\n",
+    "import openai\n",
+    "from dotenv import load_dotenv\n",
+    "from langchain.embeddings.openai import OpenAIEmbeddings\n",
+    "from langchain.schema import BaseRetriever\n",
+    "from langchain.vectorstores.azuresearch import AzureSearch"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Configure OpenAI settings\n",
+    "Configure the OpenAI settings to use Azure OpenAI or OpenAI"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Load environment variables from a .env file using load_dotenv():\n",
+    "load_dotenv()\n",
+    "\n",
+    "openai.api_type = \"azure\"\n",
+    "openai.api_base = os.getenv('OPENAI_ENDPOINT')\n",
+    "openai.api_version = \"2023-03-15-preview\"\n",
+    "openai.api_key = os.getenv(\"OPENAI_API_KEY\")\n",
+    "model: str = os.getenv('OPENAI_EMBEDDING_DEPLOYED_MODEL', \"text-embedding-ada-002\")"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Configure vector store settings\n",
+    " \n",
+    "Set up the vector store settings using environment variables:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "vector_store_address: str = os.getenv('AZURE_SEARCH_SERVICE_ENDPOINT')\n",
```

Comment:
Since this is a notebook people are using interactively, wouldn't it be better for these to be just variables that people update to the right value when trying it out?

Reply:
sure!